src/Guide/Api/Types.hs

```diff
@@ -19,10 +19,13 @@ module Guide.Api.Types
   , SearchSite(..)
   , Site(..)
   , TraitSite(..)
+
+  -- Type for backend
```

Comment:
`TraitType` is not a "type for backend", it is more-or-less exposed to the frontend in URLs. It's hard to write a good description for this type, but a misleading description is worse than none, so you can just write:

```
-- * Other types
```

Note that you need an `*` there if you want it to be rendered as a separate section in Haddocks.